content/pt/docs/concepts/signals/metrics.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Métricas
+weight: 2
+description: Uma medição capturada em tempo de execução.
+default_lang_commit: 09f39d1aef5d792a68be42746a06ac828a499350
+---
+
+Uma **métrica** é uma **medição** de um serviço em tempo de execução. O momento
+de captura dessas medições é conhecido como **evento de métrica**, que consiste
+não apenas na medição em si, mas também no momento em que ela foi capturada e os
+metadados associados.
+
+Métricas de aplicação e solicitação são indicadores importantes de
+disponibilidade e desempenho. Métricas personalizadas podem fornecer percepções
+sobre como os indicadores de disponibilidade impactam a experiência do usuário
+ou o negócio. Dados coletados podem ser usados para alertar sobre uma
+interrupção ou acionar decisões de agendamento para escalar uma aplicação
+automaticamente mediante alta demanda.
+
+Para entender como as métricas no OpenTelemetry funcionam, vamos dar uma olhada
+em uma lista de componentes que desempenharão um papel na instrumentação do
+nosso código.
+
+## Meter Provider
+
+Um Meter Provider (às vezes chamado de `MeterProvider`) é uma fábrica para
+`Meter`s (Medidas). Na maioria dos aplicativos, um Meter Provider é inicializado
+uma vez e seu ciclo de vida corresponde ao ciclo de vida do aplicativo. A
+inicialização do Meter Provider também inclui a inicialização do Resource e do
+Exporter. Normalmente, é o primeiro passo na medição com OpenTelemetry. Em
+alguns SDKs, um Meter Provider global já está inicializado para você.
```

Comment:
I'd also avoid capitalizing words, even if the original is doing it (we need to change that, IMHO). Unless it's code entities like `MeterProvider`. +CC @svrnm